index.html

```diff
@@ -216,7 +231,7 @@ <h3 class="connect">
         <div class="contact">
             <div class="text-contact">
                 <h2>Contact me</h2>
-                <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it
+                <p class="footerp">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it
                 </p>
                 <div class="form-div">
                     <form action="https://formspree.io/f/xvongqly"
```

Comment:
- [ ] According to the requirement, the form button should be aligned to the middle on the desktop version but yours is aligned to the left of the form. Kindly implement this requirement. 👍 

### Your Design

![image](https://user-images.githubusercontent.com/77106962/214151427-ee2976b5-cf59-47e2-a4f6-c7ab6f717da3.png)

### Figma Design

![image](https://user-images.githubusercontent.com/77106962/214151552-ee8e17a1-c3d0-439a-828a-5d579a915528.png)